docs/labs/README.md

```diff
@@ -64,18 +64,25 @@ In Standard GKE clusters Control Plane (master) is administered by Google so lab
 
 Underlying OS in nodes running at GKE is Container-Optimized OS by default but it could be Ubuntu also.
 
-As stated in previous platform, in the case of Ubuntu Linux as node OS we should use Ubuntu Linux as the docker image for the build step in KMM CRD as it will almost certainly contain the kernel headers package matching the node kernel version. 
+As stated in previous platform, in the case of Ubuntu Linux as node OS we should use Ubuntu Linux as the docker image for the build step in KMM CRD as it will certainly contain the kernel headers package matching the node kernel version. As of today, Ubuntu 20.04 focal LTS includes packages matching kernel version running in Ubuntu with containerd nodes at GKE. This may change as newer versions of Ubuntu will be used on GKE nodes.
 
 We could get the whole list of available linux-headers packages for GKE just by doing this:
-``
+```console
 apt update && apt-cache search linux-headers | grep gke
-``
+```
+Use of Ubuntu based nodes with containerd in GKE is the straight, preferred and easier option if you are willing to use KMM.
```

Comment:
I would introduce the paragraph that follows and emphasize that it's not recommended:
```suggestion
The default option, Google's Container-optimized OS, comes with security settings that make it difficult to load out-of-tree kernel modules.
Using Ubuntu nodes with containerd in GKE is the straight, preferred and easier option if you are willing to use KMM.
```